src/components/pages/Auth/Login/index.tsx

```diff
@@ -6,14 +6,19 @@ import Auth from '..';
 import TextField from '~components/reusables/CustomField/TextField';
 import Api from '~api';
 import authStore from '~src/store/auth';
+import useRememberMe from '~components/pages/Auth/Login/hooks-login/useRememberMe';
 
 const { api } = new Api();
 
 const Login = () => {
   const { update } = authStore((state) => state);
+  console.log(update);
```

Comment:
remove the console.log statement

Reply:
> remove the console.log statement

should i remove the update variable as well, it is not being used